### ISSUE TITLE
Dark theme.

### ISSUE THREAD

**Issue by linkjay** [RainLoop/rainloop-webmail#1720]
I understand that there are already issues relating to this, but can we have a dark theme added.

I don't mean custom themes or the "Dark squares" thing, I mean a theme that changes all the colors to be a dark sort of pallete. Basically, all colors would be dark, and there would be little to no white or anything like that.

Using rainloop at night is a sort of retina-burner for me at the moment.

**Comment by microfx** [comment]
any news on this? most important feature imho!

**Comment by R0dri** [comment]
Please, dark mode will put this web mail light years ahead

**Comment by clinkadink** [comment]
I totally agree. I use a 48" 4K TV as my monitor. When I check my mail at night .... my eyes start melting.

I have created my own **Dark Theme** for now. But hope Rainloop provide one in future. I'll post it here in case someone wants it. This obviously unsupported and you use it at your own risk ... bla, bla, bla.

Extract the CSS file in the [app.min.zip](https://github.com/RainLoop/rainloop-webmail/files/4893299/app.min.zip) file, and paste into` /rainloop/v/1.14.0/static/css/`, then refresh your browser. You may need to hold CTRL + F5 to clear any cached CSS. I didn't.

**Comment by tripslip38** [comment]
@clinkadink this is great, thanks. I found one issue which was the .hasUnseenSubMessage class. It was still using a light yellow color. I searched the file for that classname and replaced the value with #353434.

**Comment by ShadwDrgn** [comment]
I see that 4 years ago someone requested dark mode, but it's still not a feature and you ahve to use some modified css in the source to do it. Has this per chance been added elsewhere and this is a dead issue?

**Comment by the-djmaze** [comment]
@ShadwDrgn SnappyMail is a fork and has dark mode.
It also has a setting to remove (background)-color from messages so that messages are also in dark mode.

**Comment by OckhamOdyssey** [comment]
> @ShadwDrgn SnappyMail is a fork and has dark mode.
> It also has a setting to remove (background)-color from messages so that messages are also in dark mode.

Definitely SnappyMail is a better solution.
